.github/actions/detect-workflow/main.go

```diff
@@ -0,0 +1,40 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"strings"
+
+	"github.com/slsa-framework/slsa-github-generator/github"
+)
+
+func main() {
+	ctx := context.Background()
+
+	c, err := github.NewOIDCClient()
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	audience := "slsa-framework/slsa-github-generator/detect-workflow"
```

Comment:
I suspect this may fail when use from a different repo? Can we pass audience as an option?

Reply:
done